### ISSUE TITLE
Enhanced Dimensional Pockets and Enhanced Dimensional Shifter do not work.

### ISSUE THREAD

**Issue by ToshieCosed** [TheRealZeher/DimensionalPockets-II#108]
Mods included in further testing were:
Solar Flux Reborn --Latest possible.
JEI -- Latest possible.
Dimensional Pockets II --Latest possible.
Forge Version tested on: 1.19.2 -- Forge: 43.3.2 --Edited to fix error in writing this.

When uncharged while using the enhanced dimensional shifter the chat message a player gets is "you haven't linked your shifter to a pocket yet!"
When using a normal dimensional shifter on an enhanced pocket dimension it also does the same behavior.
Yet, mysteriously, the normal dimensional shifter does work on a pocket dimension (the unenhanced one), even allows linking while uncharged.
When the enhanced dimensional shifter is charged, attempting to link to an enhanced dimensional pocket does not report anything in chat and nothing appears to work.

So in summary behavior is:
Dimensional Shifter reports no link when used on an enhanced dimensional pocket while uncharged.
Enhanced Dimensional shifter reports no link when used on an enhanced dimensional pocket while uncharged.
In both above cases, while charged neither item seems to function at all or interact with the Enhanced Dimensional Pocket.
In all cases, the normal dimensional shifter ONLY seems to work properly on the normal dimensional pocket.

Expected behavior: Linking to an enhanced dimensional pocket while uncharged the dimensional shifter should allow interaction with the enhanced dimensional pocket and show proper true/false indications in chat, followed by being able to shift into and out of enhanced pocket dimensions.

**Comment by TheCosmicNebula** [comment]
This has now been fixed as of version 1.19.2-8.3.1.10
